cluster-test/test-orchestrator/stat_ingester_test.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"strings"
+)
+
+type nopWriter struct{}
+
+func (r nopWriter) Write(bts []byte) (int, error) {
+	return len(bts), nil
+}
+
+func ExampleStatIngester() {
+	si := NewStatIngester(nopWriter{})
+	scanner := bufio.NewScanner(strings.NewReader(stats2))
+	si.IngestStats(scanner)
+	fmt.Println(si.IsClusterStable(
+		[]string{"172.18.24.220:3000", "172.18.24.220:3001", "172.18.24.220:3002"},
+	))
+
+	si = NewStatIngester(nopWriter{})
+	scanner = bufio.NewScanner(strings.NewReader(stats2))
+	si.IngestStats(scanner)
+	fmt.Println(si.IsClusterStable(
+		[]string{"172.18.24.220:3000", "172.18.24.220:3001"},
+	))
+
+	// Output:
+	// false
+	// true
+}
+
+func ExampleWaitForStable() {
+	si := NewStatIngester(nopWriter{})
+	scanner := bufio.NewScanner(strings.NewReader(stats2))
+	si.IngestStats(scanner)
+	si.WaitForStable(
+		[]string{"172.18.24.220:3000", "172.18.24.220:3001"},
+	)
+
+	// Output:
```

Comment:
This example isn't "testing" anything right? Is it possibly to make it a unit test?